openedx/core/djangoapps/content/block_structure/tasks.py

```diff
@@ -21,53 +22,92 @@
 NO_RETRY_TASKS = (XMLSyntaxError, LoncapaProblemError, UnicodeEncodeError)
 
 
-@task(
-    default_retry_delay=settings.BLOCK_STRUCTURES_SETTINGS['TASK_DEFAULT_RETRY_DELAY'],
-    max_retries=settings.BLOCK_STRUCTURES_SETTINGS['TASK_MAX_RETRIES'],
-    bind=True,
-)
+def block_structure_task():
+    """
+    Decorator for block structure tasks.
+    """
+    return task(
+        default_retry_delay=settings.BLOCK_STRUCTURES_SETTINGS['TASK_DEFAULT_RETRY_DELAY'],
+        max_retries=settings.BLOCK_STRUCTURES_SETTINGS['TASK_MAX_RETRIES'],
+        bind=True,
+    )
```

Comment:
Nit: It might make sense to have `block_structure_task()` pass through a `**kwargs` to the underlying `task()` decorator, so callers can still customize functionality while still getting the new defaults.

If you think this is premature optimization, feel free to ignore.